framework/sandstone_tests.h

```diff
@@ -23,6 +23,18 @@ extern struct test __stop_tests;
 
 extern struct test mce_test;
 
+typedef enum {
+    TEST_ENABLED,
+    TEST_DISABLED,
+    TEST_NOT_FOUND,
+} test_cfg_status;
+
+struct test_cfg_info {
+    struct test *test;
+    test_cfg_status st;
+    std::string attribute;
```

Comment:
Given you have a non-trivial type here, please initialise the other two members. While you're at it, please reorder:
```c++
struct test_cfg_info
{
    enum test_status {
        disabled = 0,
        enabled = 1-,
        not_found = -1,
    };
    struct test *test = nullptr;
    std::string atribute;
    test_status status = not_found;
};
```